marconi-mamba/src/Marconi/Api/HttpServer.hs

```diff
@@ -32,11 +33,10 @@ bootstrap env =  runSettings
         (env ^. httpSettings)
         (serve (Proxy @API) (server (env ^. queryEnv ) ) )
 
-server :: DBQueryEnv -> Server API
+server :: UtxoIndexerEnv -> Server API
```

Comment:
Given that signature, if we want to add other indexers, we need to add additional parameters or should we just modify `UtxoIndexerEnv` (and rename to `DBQueryEnv` or something)?

Reply:
if we go with this design, we should add the new idexer to  [Marconi.Api.Types.UtxoIndexerWrapper ](https://github.com/kayvank/plutus-apps/blob/1314/address-utxo-indexer-new-marconi-api/marconi-mamba/src/Marconi/Api/Types.hs#L46) and change name(s) accordingly